docs/HOWTOs/rsnapshot-windows-howto

```diff
@@ -282,7 +282,7 @@ E- Restoring Files:
     b- Set the homes shares to read only in /etc/samba/smb.conf.
        [homes]
        comment = Home Directories
-       browseable = no
+       browsable = no
```

Comment:
Is it supposed to be like that - I'm not sure haven't configured any Samba in a while?